SECURITY.md

```diff
@@ -0,0 +1,110 @@
+process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal
+communication and external disclosure but will need help from relevant developers to successfully
+run this process.
+
+The initial Product Security Team will consist of members of Steering Committee and Core Development Team in the private
+[OpenTofu Security](https://groups.google.com/a/opentofu.org/g/security) list. In the future we may
+decide to have a subset of maintainers work on security response given that this process is time
+consuming.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+The OpenTofu community asks that all suspected vulnerabilities be privately and responsibly disclosed
+via the [reporting policy](README.md#reporting-security-vulnerabilities).
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY email
+[security@opentofu.org](security@opentofu.org) to inform the Product
+Security Team (PST) about the vulnerability so they may start the patch, release, and communication
+process.
+
+If possible the PST will ask the person making the public report if the issue can be handled via a
+private disclosure process (for example if the full exploit details have not yet been published). If
+the reporter denies the request for private disclosure, the PST will move swiftly with the fix and
+release process. In extreme cases GitHub can be asked to delete the issue but this generally isn't
+necessary and is unlikely to make a public disclosure less damaging.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability a member of the PST will volunteer to lead coordination with the "Fix Team"
+and is responsible for sending disclosure emails to the rest of the community. This lead will be
+referred to as the "Fix Lead."
+
+The role of Fix Lead should rotate round-robin across the PST.
+
+Note that given the current size of the OpenTofu community it is likely that the PST is the same as
+the "Fix team." (I.e., all maintainers). The PST may decide to bring in additional contributors
+for added expertise depending on the area of the code that contains the vulnerability.
+
+All of the timelines below are suggestions and assume a private disclosure. The Fix Lead drives the
+schedule using their best judgment based on severity and development time. If the Fix Lead is
+dealing with a public disclosure all timelines become ASAP (assuming the vulnerability has a CVSS
+score >= 4; see below). If the fix relies on another upstream project's disclosure timeline, that
+will adjust the process as well. We will work with the upstream project to fit their timeline and
+best protect our users.
+
+### Fix Team Organization
+
+These steps should be completed within the first 24 hours of disclosure.
```

Comment:
To ensure an SLA of 24h, we'd have to have on-call rotations and a bunch of other stuff that sounds somewhat impractical for an open source foundation? At least, until we have a fully paid team working on this?

Reply:
I've updated it to 72 hours. I think it's reasonable across the entire Core team and SC members. We can always update it in the future. It's handled via a mailing list, so I don't think we should have an on-call rotations for this matter (at least right now).

Reply:
I guess I'm thinking primarily of the holidays: e.g., a security report comes in during Christmas... Are we sure someone will be around to respond promptly?

Reply:
I guess to put it another way: do we want to commit, publicly and in writing, to specific SLAs/SLOs at this stage in the project? We will certainly react to security issues as quickly as we possibly can, but do we want to make that a hard commitment at this stage, before full-time, dedicated people are even hired?

Reply:
@brikis98 I agree, you right. I've removed sections with SLO / timelines for now.